src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -102,7 +101,7 @@
                     + "(?<tagArguments>(?: t/[^ ]+)*)"); // variable number of tags
     
     private static final Pattern RECURRING_TASK_DATA_ARGS_FORMAT = 
-            Pattern.compile("(?<recurring>\\b(?i)daily|weekly|monthly|yearly(?i)\\b)");
+            Pattern.compile("(?<recurring>\\b(?i)daily|weekly|monthly|yearly|none(?i)\\b)");
```

Comment:
I extend edit's functionality so it now can set a recurring task back to a non-recurring one if needed